recipe/meta.yaml

```
@@ -1,52 +1,100 @@
     - smac = smac.smac_cli:cmd_line_call
-  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
+    - python
     - pip
-    - python >=3.8
+    - setuptools
+    - wheel
   run:
-    - configspace >=0.6.1
-    - dask-core
-    - dask-jobqueue
-    - regex
-    - numpy >=1.23.3
-    - scipy >=1.9.2
+    - python
+    - numpy >=1.7.1
+    - scipy >=1.7.0
     - psutil
-    - pynisher >=1.0.0
-    - ConfigSpace >=0.6.1
+    - pynisher >=0.4.1
+    - configspace >=0.4.14,<0.5
     - joblib
-    - scikit-learn >=1.1.2
-    - pyrfr >=0.9.0
-    - dask
+    - scikit-learn >=0.22.0
+    - pyrfr >=0.8.0
+    - dask-core
+    - distributed
     - emcee >=3.0.0
-    - python >=3.8
-    - pyyaml
-    - __linux
+
+{% set tests_to_ignore = "" %}
+# ignore test_cli because they require examples folder and run scripts
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_cli/test_restore_state.py" %}
+# ignore test_psmac_facade.py and test_hydra_facade.py because they
+# use a non existing data file (spear_hydra_test_scenario.txt).
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_facade/test_hydra_facade.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_facade/test_psmac_facade.py" %}
+# ignore tests that crash instead of timeout
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_tae/test_exec_func.py" %}  # [osx]
+
+{% set tests_to_skip = "" %}
+# skip test_gp_on_sklearn_data because using load_boston deprecated in sklearn
+{% set tests_to_skip = tests_to_skip + "test_rf_on_sklearn_data or test_gp_on_sklearn_data" %}
+# skip tests problematic with multiprocessing and pickle
+{% set tests_to_skip = tests_to_skip + " or test_incorporate_run_results_callback_stop_loop or test_update_intensification_percentage" %}  # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_parameter_order or test_check_deterministic_rosenbrock" %}  # [osx]
+# skip tests for numerical approximation problems on osx
+{% set tests_to_skip = tests_to_skip + " or test_log_runtime_with_imputation or test_cost_with_imputation" %}  # [osx]
+# skip tests because configuration is "{ 'a': 0, 'b': 100, }" instead of "{ 'a': 100, 'b': 0, }"
+# which is strange, but hard to guess why (again approximation?).
+{% set tests_to_skip = tests_to_skip + " or test_eval_challenger" %}  # [osx]
+# skip tests for numerical approximation problems on linux-64
+{% set tests_to_skip = tests_to_skip + " or test_lnprob_and_grad_scalar" %}  # [linux and x86_64]
+# skip tests for numerical approximation problems on osx-64
+{% set tests_to_skip = tests_to_skip + " or test_facades" %}  # [osx and x86_64]
+# skip tests that are way too slow on osx-arm64
+{% set tests_to_skip = tests_to_skip + " or test_AC or test_facades" %}  # [arm64]
 
 test:
+  source_files:
+    - gh_src/test
   imports:
     - smac
-
+  requires:
+    - pip
+    - pytest
```

```suggestion
    - pytest >=4.6
```
https://github.com/automl/SMAC3/blob/v1.2.0/extras_require.json#L10C2-L10C2

---

ok added